### DISCUSSION TITLE
Add git tag (git describe --tags) to the environment

### DISCUSSION THREAD

**Discussion by graup** [vercel/community#166] [Idea]
Hi! At the moment, the exposed git related environment variables do not contain any info about tags. ([docs](https://vercel.com/docs/concepts/git/vercel-for-github#configuring-vercel-for-github))

I propose adding `VERCEL_GIT_TAG` with the value of `git describe --tags`.

This would make my build workflow much easier. We're using the tag to define the release version for Sentry etc.

This was also mentioned [here](https://github.com/vercel/vercel/issues/245#issuecomment-828016605) but I thought I'd open a new issue for better visibility.

**Comment by Pieparker** [comment]
Hey there, we've moved this issue from the Vercel CLI Discussion to the Vercel Community Discussions as a feature suggestion. Thank you!

**Reply by srosato** [reply]
Could you provide the link to that discussion please?

**Reply by graup** [reply]
It's this discussion. They're saying they moved it here :)

**Reply by srosato** [reply]
Oops my bad :joy:

**Comment by HesselTjeerdsma** [comment]
Would like to +1 as well, would be very useful for us to do so directly from Vercel!

**Comment by andrealettieri80** [comment]
We really need this!! +1 :)

**Comment by Exelord** [comment]
We need it too <3

**Comment by eclipse313** [comment]
Is also desperately needed by me. I want to query the current day to be able to display it in my Vue project.

**Comment by Burry** [comment]
Same use-case here, would like to use tags to generate a semver build ID used across our monitoring tools. And right now that requires weird hacks if you want to deploy with Vercel
